packages/shared/components/popups/ActivityDetailsPopup.svelte

```diff
@@ -48,41 +50,29 @@
             value = transactionPayload.data.essence.data.value
         }
     }
-    $: currencyValue = convertToFiat(
-        value,
-        $currencies[CurrencyTypes.USD],
-        $exchangeRates[$activeProfile?.settings.currency]
-    )
 
+    let transactionSubjectAddressOrAccount: { isSubjectAccount: boolean; subject: string }
     $: transactionSubjectAddressOrAccount = getTransactionSubjectAddressOrAccount(transactionPayload)
```

Comment:
Same as before, you don't need to initialize this, you just need to return the correct type from `getTransactionSubjectAddressOrAccount(transactionPayload)`

Reply:
Removed in next PR